pkg/testgrid/testgrid_test.go

```diff
@@ -56,7 +57,7 @@ func TestXMLOutput(t *testing.T) {
 `
 
 	// Create a test file
-	if err := CreateXMLOutput(tc, name); err != nil {
+	if err := pkgTestgrid.CreateXMLOutput(tc, name); err != nil {
```

Comment:
This test should like in pkg repo

Reply:
It's already in https://github.com/knative/pkg/blob/master/test/testgrid/testgrid_test.go, I just forgot to delete it here.

Deleted.